web/satellite/src/components/projectsList/ProjectsList.vue

```diff
@@ -0,0 +1,180 @@
+                <VPagination
+                    :total-page-count="projectsPage.pageCount"
+                    :on-page-click-callback="onPageClick"
+                />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+import VButton from '@/components/common/VButton.vue';
+import VList from '@/components/common/VList.vue';
+import VPagination from '@/components/common/VPagination.vue';
+import ProjectsListItem from '@/components/projectsList/ProjectsListItem.vue';
+import SortProjectsListHeader from '@/components/projectsList/SortProjectsListHeader.vue';
+
+import { RouteConfig } from '@/router';
+import { ACCESS_GRANTS_ACTIONS } from '@/store/modules/accessGrants';
+import { BUCKET_ACTIONS } from '@/store/modules/buckets';
+import { PAYMENTS_ACTIONS } from '@/store/modules/payments';
+import { PROJECTS_ACTIONS } from '@/store/modules/projects';
+import { Project, ProjectsCursor, ProjectsPage } from '@/types/projects';
+import { PM_ACTIONS } from '@/utils/constants/actionNames';
+import { LocalData } from '@/utils/localData';
+
+const {
+    FETCH_OWNED,
+} = PROJECTS_ACTIONS;
+
+@Component({
+    components: {
+        SortProjectsListHeader,
+        VButton,
+        VList,
+        VPagination,
+    },
+})
+export default class Projects extends Vue {
+
+    private currentPageNumber: number = 1;
+
+    private currentProjectsPage: ProjectsPage = new ProjectsPage();
+
+    private FIRST_PAGE = 1;
+
+    /**
+     * Lifecycle hook after initial render where list of existing ownded projects is fetched.
+     */
+    public async mounted(): Promise<void> {
+        try {
+            await this.$store.dispatch(FETCH_OWNED, this.currentPageNumber);
+        } catch (error) {
+            await this.$notify.error(`Unable to fetch owned projects. ${error.message}`);
+        }
+    }
+
+    /**
+     * Fetches owned projects page page by clicked page number.
+     * @param index
+     */
+    public async onPageClick(page: number): Promise<void> {
+        this.currentPageNumber = page;
+        try {
+            await this.$store.dispatch(FETCH_OWNED, this.currentPageNumber);
+        } catch (error) {
+            await this.$notify.error(`Unable to fetch owned projects. ${error.message}`);
+        }
+    }
+
+    /**
+     * Returns ProjectsList item component.
+     */
+    public get itemComponent() {
+        return ProjectsListItem;
+    }
+
+    /**
+     * Redirects to create project page.
+     */
+    public onCreateClick(): void {
+        this.$router.push(RouteConfig.CreateProject.path);
+    }
+
+    /**
+     * Returns projects page from store.
+     */
+    public get projectsPage(): ProjectsPage {
+        return this.$store.state.projectsModule.page;
+    }
+
+    /**
+     * Fetches all project related information.
+     * @param project
+     */
+    public async onProjectSelected(project: Project): Promise<void> {
```

Comment:
I think this should also navigate to the main dashboard page. When I test it, clicking on the project changes the project, but there is no visual feedback suggesting that anything happened. If we navigate, it will let the user know.